docs/src/main/asciidoc/_admin-guide/management-tasks/Suspend,_Resume_and_Graceful_shutdown.adoc

```diff
@@ -232,3 +232,42 @@ the proper way to control the lifecycle of a domain mode server process is via t
 management API and its managing Host Controller, not via direct signals to the server
 process.
 
+[[non-graceful-startup]]
+== Non-graceful Startup
+
+By default, Wildfly starts up gracefully, meaning that incoming requests are queued until the
```

Comment:
s/queued/queued or cleanly rejected/g

The queuing behavior is limited to HTTP requests hitting the undertow listener, and only if queuing is enabled on that listener.

Reply:
Also

s/Wildfly/WildFly